Sources/PenguinTesting/StdlibProtocolTests.swift

```diff
@@ -280,10 +271,6 @@ extension Sequence where Element: Equatable {
       t.checkSemantics()
     }
     else { checkSequenceSemantics(expecting: expectedContents) }
-    
-    if let t = MutableCollectionChecker(self, expecting: Array(self)) as? Checker {
-      t.checkSemantics()
-    }
```

Comment:
You simply dropped automatic checking of mutable collection declared conformances?  Seems to me that you should either not do that, or we should revert my commit(s) that added this functionality on the argument that it isn't needed.

Reply:
Let's revert it. I'll make a commit doing that.

Reply:
Next time, a better move might be to go to the PR and use the revert button.  Again, it keeps separate things separate.